src/app/modules/customer-management/components/projects-type/components/create-project-type/create-project-type.component.spec.ts

```diff
@@ -20,6 +21,7 @@ describe('InputProjectTypeComponent', () => {
   let projectTypeIdToEditMock;
   let allProjectTypesMock;
   let getProjectTypeByIdMock;
+  let getProjectTypeByIdSelectorMock;
```

Comment:
tha same case. you can use getProjectTypeByIdMock and not create a new variable.